src/mpyl/schema/mpyl_config.schema.yml

```diff
@@ -176,6 +176,9 @@ definitions:
       artifactRepository:
         description: "The repository where the build artifacts are persisted"
         "$ref": "#/definitions/Git"
+      argoRepository:
+        description: "The argocd repository that contains the kubernetes manifests"
+        "$ref": "#/definitions/Git"
```

Comment:
Artifact repository was meant to cover both build cache and manifests. I understand you need to be able to store them in different repositories.
For the naming perhaps you could follow the choices here https://github.com/Vandebron/mpyl/blob/32476643940c8daa4bc38658170e2a937c24ff8e/src/mpyl/cli/build.py#L489

Reply:
We decided on `artifactRepository` -> `cachingRepository` & `argoRepository` -> `artifactRepository`

Reply:
Why `artifactRepository`? It's not storing any artifacts AFAICS. Rather it stores manifests.

Reply:
Well we picked kubectl manifests for now, but you could store whatever build artifact you want.

Reply:
Indeed, you could also use HELM charts. But in that case would you call these artifacts or manifests? I would call them manifests. Artifacts contain executable code, like .jars or docker containers.